homesensorhub/sensors/sensor.py

```diff
@@ -14,6 +14,20 @@ def __init__(self, sensor, name):
         self.__sensor = sensor
         self.__name = name
 
+    def build_sensor_packet(self, type, value, timestamp, measurement):
+        """Build a packet with information taken from each sensor."""
+        print(self.__name)
```

Comment:
remove debug print

Reply:
Solved in 92d88406c2998f664c3f53a9cca80e3f51d92814.